addons/event/views/event_registration_views.xml

```diff
@@ -184,28 +176,26 @@
                             </a>
                         </div>
                     </t>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="#{!record.active.raw_value ? 'oe_kanban_card_ribbon' : ''} oe_kanban_global_click o_event_registration_kanban container-fluid">
+                    <t t-name="kanban-card" class="ps-0 pe-0">
```

Comment:
the two  templates above must be reworked to...

Reply:
I cannot find any suitable changes in the templates since Bootstrap classes are used, and each class has a specific purpose in the UI. If you can guide me regarding any particular changes, it would be helpful.

Reply:
my bad, you're right, sorry